bpo-45132 Remove deprecated __getitem__ methods (#28225)
python/cpython: Doc/library/wsgiref.rst

.. class:: FileWrapper(filelike, blksize=8192)

   A wrapper to convert a file-like object to an :term:`iterator`.  The resulting objects

>    support the :meth:`__iter__` iteration style. As the object is iterated over, the

Reviewer: ```suggestion
   are :term`iterable`\ s. As the object is iterated over, the
```

      for chunk in wrapper:
          print(chunk)

>    .. deprecated-removed:: 3.8 3.11

Reviewer: ```suggestion
   .. versionchanged:: 3.11
```

>       Support for sequence protocol :meth:`__getitem__` has been removed.

Reviewer: ```suggestion
      Support for :meth:`__getitem__` method has been removed.
```
